tests/conftest.py

```diff
@@ -1507,10 +1507,12 @@ def site_builder_data_context_with_html_store_titanic_random(
     context.profile_datasource("random")
     context.profile_datasource(context.list_datasources()[0]["name"])
 
-    context._project_config.anonymous_usage_statistics = {
-        "enabled": True,
-        "data_context_id": "f43d4897-385f-4366-82b0-1a8eda2bf79c",
-    }
+    context.variables.config.anonymous_usage_statistics = (
```

Comment:
why is this assigned through the config attribute and not `context.variables.anonymous_usage_statistics`?